src/idea.js

```diff
@@ -12,6 +12,6 @@ class Idea {
 
   }
   updateIdea() {
-
+    this.star = !this.star;
```

Comment:
This is fantastic, I think we should try to move a few of our other functions (like deleteCard) into these methods that we were provided in Iteration 0.  I'll work on that a bit today!